clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -501,6 +532,10 @@ test_template_with_repstr(void *dr_context)
                   << "\n";
         return 1;
     }
+    assert(!syscall_stats.syscall_errno_counts.empty());
+    assert(syscall_stats.syscall_errno_counts[EINVAL] = 2);
```

Comment:
test_template_with_repstr is X86 only. Would be better to move this to a test that's run on all env, or its own separate test.

Reply:
Moved to a helper check_syscall_stats() and added a call in test_trace_templates() too.